Tools/clinic/clinic.py

```diff
@@ -1401,24 +1426,10 @@ def parser_body(
                 nargs = f"Py_MIN(nargs, {max_pos})" if max_pos else "0"
 
             if limited_capi:
-                # positional-or-keyword arguments
```

Comment:
It was moved down, to a fallback code used if inlining failed. For now, it always failed for limited_capi, because there is no public variant of `_PyArg_UnpackKeywords`.

Reply:
Thanks for the explanatory review comments.